sequence_processing_pipeline/templates/nuqc_job.sh

```diff
@@ -93,123 +87,64 @@ function mux-runner () {
-            --adapter_fasta {{knwn_adpt_path}} \
-            --html {{html_path}}/${html_name} \
-            --json {{json_path}}/${json_name} \
-            --out1 ${r1_filt} \
-            --out2 ${r2_filt}
+            -w 16 \
+            --adapter_fasta /home/qiita_test/qiita-spots/qp-knight-lab-processing/fastp_known_adapters_formatted.fna \
+            --html ${WKDIR}NuQCJob/fastp_reports_dir/html/${html_name} \
+            --json ${WKDIR}NuQCJob/fastp_reports_dir/json/${json_name} \
+            --stdout > ${r1_filt}
 
         # multiplex and write adapter filtered data all at once
         cat ${r1_filt} | \
             sed -r "1~4s/^@(.*)/@${i}${delimiter}\1/" \
             >> ${seqs_r1} &
-        cat ${r2_filt} | \
-            sed -r "1~4s/^@(.*)/@${i}${delimiter}\1/" \
-            >> ${seqs_r2} &
         cat ${r1_filt} | \
             gzip -c > ${r1_adapter_only} &
-        cat ${r2_filt} | \
-            gzip -c > ${r2_adapter_only} &
         wait
 
         rm ${r1_filt} &
-        rm ${r2_filt} &
         wait
     done
-}
-export -f mux-runner
-
-function minimap2-runner () {
-    if [[ -z ${fcurrent} ]]; then
-        echo "fcurrent not set"
-        exit 1
-    fi
-
-    jobd=${TMPDIR}
-
-    id_map=${jobd}/id_map
-    if [[ ! -f ${id_map} ]]; then
-        echo "No samples..."
-        return
-    fi
-
-    seqs_r1=${jobd}/seqs.r1.fastq
-    seqs_r2=${jobd}/seqs.r2.fastq
-    seqs_mmpe_r1=${jobd}/seqs.mmpe.r1.fastq
-    seqs_mmpe_r2=${jobd}/seqs.mmpe.r2.fastq
-    seqs_mmpese=${jobd}/seqs.mmpese.fastq
-    seqs_mmpese_r1=${jobd}/seqs.mmpese.r1.fastq
-    seqs_mmpese_r2=${jobd}/seqs.mmpese.r2.fastq
-
-    # PE operation
-    minimap2 -2 -ax sr -t 12 \
-        ${fcurrent} \
-        ${seqs_r1} \
-        ${seqs_r2} | \
-            samtools fastq \
-                -@ 4 \
-                -f 12 \
-                -F 256 \
-                -N \
-                -1 ${seqs_mmpe_r1} \
-                -2 ${seqs_mmpe_r2}
-
-    rm ${seqs_r1} &
-    rm ${seqs_r2} &
-    # no need to block
-
-    # SE operation
-    # run r1/r2 serially to avoid minimap2 picking up on interleve
-    minimap2 -2 -ax sr -t 12 \
-        ${fcurrent} \
-        <(cat ${seqs_mmpe_r1} ${seqs_mmpe_r2}) | \
-            samtools fastq \
-                -@ 4 \
-                -f 4 \
-                -F 256 \
-                -0 ${seqs_mmpese}
-
-    rm ${seqs_mmpe_r1} &
-    rm ${seqs_mmpe_r2} &
-    # no need to block
-
-    {{splitter_binary}} ${seqs_mmpese} ${seqs_mmpese_r1} ${delimiter} ${r1_tag} &
-    {{splitter_binary}} ${seqs_mmpese} ${seqs_mmpese_r2} ${delimiter} ${r2_tag} &
-    wait
-
-    rm ${seqs_mmpese} &
 
-    fastq_pair -t 50000000 ${seqs_mmpese_r1} ${seqs_mmpese_r2}
-    rm ${seqs_mmpese_r1}.single.fq &
-    rm ${seqs_mmpese_r2}.single.fq &
-    rm ${seqs_mmpese_r1} &
-    rm ${seqs_mmpese_r2} &
+    minimap2 -2 -ax sr -t 16 /scratch/databases/minimap2/human-pangenome/human-GCA-phix-db.mmi ${jobd}/seqs.r1.fastq -a | \
```

Comment:
Is the plan to use these as a list from the configuration file? Remember that we need 2 mmi for WGS and 3 mmi for MTX.